src/error.rs

```diff
@@ -0,0 +1,62 @@
+use std::borrow::Cow;
+use std::error;
+use std::fmt;
+use std::io;
+
+/// Represents an error encountered while parsing an MO file.
+#[derive(Debug)]
+pub enum Error {
+    /// An incorrect magic number has been encountered
+    BadMagic,
+    /// An invalid byte sequence for the given encoding has been encountered
+    DecodingError,
+    /// An unexpected EOF occured
+    Eof,
+    /// An I/O error occured
+    Io(io::Error),
+    /// Incorrect syntax encountered while parsing the meta information
+    MalformedMetadata,
+    /// Meta information string was not the first string in the catalog
+    MisplacedMetadata,
+    /// Invalid Plural-Forms metadata
+    PluralParsing,
+    /// An unknown encoding was specified in the metadata
+    UnknownEncoding,
+}
+use Error::*;
```

Comment:
Looks like 1.31.0 is peculiar about imports. https://travis-ci.org/github/justinas/gettext/jobs/737549423

This seems to help
```suggestion
use self::Error::*;
```